saleor/graphql/core/types/common.py

```diff
@@ -77,3 +77,15 @@ def get_adjusted(image, alt, size, rendition_key_set, info):
             url = image.url
         url = info.context.build_absolute_uri(url)
         return Image(url, alt)
+
+
+class PriceInput(graphene.InputObjectType):
```

Comment:
If we call below `DateRangeInput`, maybe this one could be `PriceRangeInout`? Or the previous one just `DateInput`? Would be more consistent IMO.